app/controllers/account/companies_controller.rb

```diff
@@ -1,5 +1,8 @@
+require 'bing_api'
+
 class Account::CompaniesController < ApplicationController
   before_action :authenticate_user!
+  #after_action Bing_api_v7::bing_pages_to_model(current_company.id), only: :create, if: -> { true }
```

Comment:
remove this garbage from here

Reply:
yeah, forgot for a while. will be removed